ArduPlane/AP_Arming.cpp

```diff
@@ -104,6 +104,12 @@ bool AP_Arming_Plane::pre_arm_checks(bool display_failure)
         ret = false;
     }
 
+    RC_Channel *chan = rc().find_channel_for_option(RC_Channel::AUX_FUNC::DISARM);
+    if (chan != nullptr && chan->get_aux_switch_pos() == RC_Channel::aux_switch_pos_t::HIGH){
```

Comment:
Should this be != LOW instead?

Reply:
Disarm is triggered when HIGH.
!= LOW means arming fails by MIDDLE position.
I think it's better to unify with HIGH.